vmdb/app/models/miq_ems_metrics_collector_worker.rb

```diff
@@ -14,7 +14,6 @@ def self.validate_config_settings(configuration = VMDB::Config.new("vmdb"))
     new_path = [:workers, :worker_base, :queue_worker_base, :ems_metrics_collector_worker]
     configuration.merge_from_template_if_missing(*new_path)
 
-    new_collector_worker_settings = configuration.config.fetch_path(*new_path)
```

Comment:
I'll take a look, see if history digs up any reasons why this was done... alas, this stuff is painful because you have to understand why it's a bug.